schedule/cryptlvm_minimal_x_staging.yaml

```diff
@@ -0,0 +1,69 @@
+---
+name:           cryptlvm_minimal_x
+description:    >
+  Combination of "cryptlvm" and "minimal_x".
+  (crypt-)LVM installations can take longer,
+  especially on non-x86_64 architectures.
+vars:
+  ENCRYPT: 1  
+  LVM: 1
+  MAX_JOB_TIME: 14400
+conditional_schedule:
+   sle15accept_licence:
+    VERSION:
+      15-SP1:
+        - installation/accept_license
+   sle15partitioning:
+    VERSION:
+      15-SP1:
+        - installation/partitioning
```

Comment:
As I can see from the Staging Test Suites, both Sle12 and Sle15 scenarios have 'partitioning' step. Could you please check this.

Please, see the examples: [SLE 15](https://openqa.suse.de/tests/2809798#step/partitioning/1), [SLE 12](https://openqa.suse.de/tests/2813989)

Reply:
seems you are correct again. fixed